File: 1234/Js/main.js
const fs = require("fs");

function toUpper(line) {
  return line.toUpperCase();
}
function toLower(line) {
  return line.toLowerCase();
}

function dance(line) {
  const result = [];
  let count = 0;
  const mapper = [toUpper, toLower];
  const n = line.length;
  for (let i = 0; i < n; i++) {
    const ch = line[i];
    if (ch.match(/[a-zA-Z]/)) {
      result.push(mapper[count % 2](ch));
      count++;
    } else {
      result.push(ch);
    }
  }
  return result.join("");
}

const data = fs.readFileSync("/dev/stdin", "utf8");
process.stdout.write(data.toLowerCase().split("\n").map(dance).join("\n"));

File: 1234/Rust/main.rs
// https://www.urionlinejudge.com.br/judge/en/problems/view/1234
use std::io::{self, BufRead, BufReader};

fn main() {
    let result = BufReader::new(io::stdin())
        .lines()
        .map(|line| line.expect("Expect a line"))
        .map(|line| {
            let mut count = 0;
            let mut result: Vec<String> = Vec::new();

            for ch in line.chars() {
                if ch.is_alphabetic() {
                    result.push(if count % 2 == 0 {
                        ch.to_uppercase().to_string()
                    } else {
                        ch.to_lowercase().to_string()
                    });
                    count += 1;
                } else {
                    result.push(ch.to_string());
                }
            }

            result.join("")
        })
        .collect::<Vec<String>>()
        .join("\n");

    print!("{}", result);
}
